src/styles/mixins/list/_list-definitions.scss

```diff
@@ -90,35 +94,47 @@ $semantic-color: (
 
 @mixin fd-list-item-states() {
   @include fd-hover() {
-    background: var(--sapList_Hover_Background);
+    @include fd-list-item-hover-state();
   }
 
   @include fd-selected() {
     @include fd-list-item-selected-state();
   }
 
   @include fd-active() {
-    background: var(--sapList_Active_Background);
+    @include fd-list-item-active-state();
+  }
+}
 
-    @include fd-hover() {
-      background: var(--sapList_Active_Background);
-    }
+@mixin fd-list-item-active-state() {
```

Comment:
> * Fiori, Horizon, HC. Focus on the expanders look quite different from all other elements. Also state active is missing. However i didn't find how it should look like, probably we should ask designers about it.
> 
> <img alt="image" width="97" src="https://user-images.githubusercontent.com/20265336/167389541-2ff44908-df94-4317-8859-4e639d343e6d.png">
> 
> <img alt="image" width="101" src="https://user-images.githubusercontent.com/20265336/167389370-434d08c4-67c3-4d2b-a2c8-fb65b93774c1.png">

I think most correct way would be if those expanders had additionally `fd-button fd-button--transparent` classes, so that they inherit ordinary button focus and other stuff, then expander would just add spacing